### PR TITLE
Add bedops to get binning

### PR DESCRIPTION
**Pull request by yuki-aoi**
Added bedops commands to generate 20 nt binning tracks by sum of single-nt read counts. bedops binary files are currently in yac9689 user directory. Removed commands of generating whole-reads tracks.

### CODE REVIEW

File: getNormalizationScripts.sh [hunk 1]
```diff
@@ -105,34 +99,46 @@ do
...
-	echo "mv ./spikein_bedgraph/${baseName}.wr36.sn.minus.bedgraph ./spikein_bedgraph/${baseName}.wr36.n.minus.bedgraph" >> $file
-
-        echo "" >> $file
         echo "# Normalize single nucleotide read counts on the minus strand by spike-in with negative values" >> $file
         echo "awk 'BEGIN{\"\t\"}{print \$1,\$2,\$3,\$4*-1*${align[1]}/${align[$i]} > \"./spikein_bedgraph/${baseName}.sn.n.minus.bedgraph\"}' ./alignments/${baseName}.sn.minus.bedgraph" >> $file
 	echo "sort -k1,1 -k2,2n ./spikein_bedgraph/${baseName}.sn.n.minus.bedgraph > ./spikein_bedgraph/${baseName}.sn.sn.minus.bedgraph" >> $file
 	echo "mv ./spikein_bedgraph/${baseName}.sn.sn.minus.bedgraph ./spikein_bedgraph/${baseName}.sn.n.minus.bedgraph" >> $file
 
+	echo "" >> $file
+	echo "#generate 20bp bin with sum of single nucleotide read counts on the plus strand" >> $file
```
Comment (ozarkp17) [inline getNormalizationScripts.sh]: If the reads are 16-36bp long, why bin over 20 bp? Should the binning be a lower percentage of the length of the read you are processing? Perhaps 5bp?

File: getNormalizationScripts.sh [hunk 2]
```diff
@@ -81,22 +81,16 @@ do
...
-        echo "" >> $file	
-	echo "# Swap +/- strand then generate strand-specific bedgraph files with whole-reads (16-36bp)" >> $file
-	echo "bedtools genomecov -ibam ./alignments/${baseName}.36.hg19.bam -bg -strand - > ./alignments/${baseName}.wr36.plus.bedgraph #forward strand" >> $file
-	echo "bedtools genomecov -ibam ./alignments/${baseName}.36.hg19.bam -bg -strand + > ./alignments/${baseName}.wr36.minus.bedgraph #reverse strand" >> $file
-
+        echo "" >> $file
 	echo "" >> $file
 	echo "# Swap +/- strand then generate strand-specific bedgraph files with single-nucleotide resolution" >> $file
-  	echo "bedtools genomecov -ibam ./alignments/${baseName}.36.hg19.bam -bg -5 -strand - > ./alignments/${baseName}.sn.plus.bedgraph #forward strand" >> $file
+  	echo "bedtools genomecov -ibam ./alignments/${baseName}.36.hg19.bam -bg -5 -strand - > ./alignments/${baseName}.sn.plus.bedgraph & #forward strand" >> $file
```
Comment (ozarkp17) [inline getNormalizationScripts.sh]: Put a space after the '#' characters in your comments. Like '# forward strand'  and '# reverse strand'

Comment (yuki-aoi) [reply getNormalizationScripts.sh]: Thanks for review and modify it! I've tested the pipeline before send the pull request and it worked.

In terms of binning, bedops spilits genome into 20 nucleotide bins, then grab the total number of "single-nucleotide resolution" values in every 20 nucleotide bin from spike-in normalized bedgraph. bedops works after extacting single-nucleotide bedgraph, that's why the length of reads (16-36 bp) does not matter. Now binned tracks are much better than whole-reads tracks and we should use binned tracks to generate heatmaps or metagene analysis. Binning by 20 nt is an example, so we will see which size of bin is good for downstream analysis.

Comment (ozarkp17) [reply getNormalizationScripts.sh]: Sounds good!

**Pull request merged**